src/store/modules/search/actions.js

```diff
@@ -15,7 +17,19 @@ export const clearSearchedResults = async ({ commit, dispatch, getters }) => {
 }
 
 export const updateSearchedValue = async ({ commit, dispatch, state }, value) => {
-  value = String(value).replace(/[\W_]+/g, '')
+  if (value.match(/.rsk/)) {
+    try {
+      const address = await getAddr(value)
```

Comment:
We should update this even when the domain is searched via address. The owner could have changed it.

Reply:
Dismiss this one. This is stored in memory. I thought we were storing it permanently…